ipfs-pinning-service.yaml

```diff
@@ -14,46 +16,69 @@ The IPFS Pinning Service API is intended to be an implementation-agnostic API&#x
 - For use in client mode by IPFS nodes and GUI-based applications
 
 
-## This spec is a work in progress! 🏗️
+> **Note**: while ready for implementation, this spec is still a work in progress! 🏗️
 
-**Your input and feedback are welcome and valuable as we develop this API spec.  
+> **Your input and feedback are welcome and valuable as we develop this API spec.
 
-Please join the design discussion at [github.com/ipfs/pinning-services-api-spec](https://github.com/ipfs/pinning-services-api-spec).**
+> Please join the design discussion at [github.com/ipfs/pinning-services-api-spec](https://github.com/ipfs/pinning-services-api-spec).**
 
 
-# The pin object lifecycle
+# Schemas
 
+This section describes the most important object types and conventions.
 
-## Creating a new pin object
 
-The user creates a pin object via `POST /pins` and receives a `PinStatus` response:
+Full list of fields and schemas can be found in `schemas` section of the YAML file.
 
-- `id` in `PinStatus` is `cid-of-pin-object`, which can can be used for modifying and/or removing the pin in the future
+## Pin and its status
```

Comment:
```suggestion
## Pin objects and pin status objects
```

Reply:
@jessicaschilling I'm afraid its bit too long and gets wrapped in the menu, anything shorter we can use? Maybe just "Pin objects" ?

> ![2020-07-23--18-26-32](https://user-images.githubusercontent.com/157609/88312218-2873bb00-cd12-11ea-9645-dce2a8682396.png)

Reply:
"Objects" is great, thanks!